packages/in_app_purchase/in_app_purchase_android/lib/src/in_app_purchase_android_platform_addition.dart

```diff
@@ -14,30 +14,35 @@ class InAppPurchaseAndroidPlatformAddition
     extends InAppPurchasePlatformAddition {
   /// Creates a [InAppPurchaseAndroidPlatformAddition] which uses the supplied
   /// `BillingClient` to provide Android specific features.
-  InAppPurchaseAndroidPlatformAddition(this._billingClient) {
-    assert(
-      _enablePendingPurchase,
-      'enablePendingPurchases() must be called when initializing the application and before you access the [InAppPurchase.instance].',
-    );
-
-    _billingClient.enablePendingPurchases();
-  }
+  InAppPurchaseAndroidPlatformAddition(this._billingClient);
 
   /// Whether pending purchase is enabled.
   ///
+  ///  **Deprecation warning:** it is no longer required to call
+  /// [enablePendingPurchases] when initializing your application. From now on
+  /// this is handled internally and the [enablePendingPurchase] property will
+  /// always return `true`.
+  ///
+  // ignore: deprecated_member_use_from_same_package
   /// See also [enablePendingPurchases] for more on pending purchases.
-  static bool get enablePendingPurchase => _enablePendingPurchase;
-  static bool _enablePendingPurchase = false;
+  @Deprecated(
+      'The requirement to call `enablePendingPurchases()` has become obsolete since Google Play no longer accepts app submissions that don\'t support pending purchases.')
+  static bool get enablePendingPurchase => true;
```

Comment:
I wasn't entirely sure about this update (always returning `true` instead of returning the `_enablePendingPurchase` variable). However I decided to do the update like this to prevent false expectations where people might think pending purchases are not enabled (because they never called `enablePendingPurchases()`).

Reply:
This makes sense to me.